src/index.html

```diff
@@ -8,10 +8,172 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post"
+      class="main__form"
+    >
+
+      <fieldset class="fieldset">
+        <legend> Personal information </legend>
+
+        <label class="form-field">
+          Surname:
+          <input
+            type="text"
+            class="enter"
+            name="surname"
+            autocomplete="off"
+            required
+          >
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input
+           type="text"
+           class="enter"
+           name="name"
+           autocomplete="off"
+           required>
```

Comment:
[Checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) 3rd item. You should fix this closing bracket issue for all of your inputs
```suggestion
          <input
           type="text"
           class="enter"
           name="name"
           autocomplete="off"
           required
         >
```